### PR TITLE
Added files via upload

### PR DESCRIPTION
**Pull request by SkylarWilliams**

### CODE REVIEW

**Comment by SkylarWilliams** [comment]
Hope this works

File: week1/SkylarWilliams [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<html>
```
Comment (yeeyang) [inline week1/SkylarWilliams]: Don't forget the `<!doctype>` at the top of the document.

File: week1/SkylarWilliams [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<head> 
+<title> Homework 1 </title>
+</head> 
+<body> 
+<h2>What does HTML stand for?</h2>
```
Comment (yeeyang) [inline week1/SkylarWilliams]: Don't forget to use a `<h1>` tag on the page before you get into your later `<h2>` tags.

File: week1/SkylarWilliams [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+<p>Hypertext Markup Language</p>
+<h2>What does CSS stand for?</h2>
+<p>Cascading Style Sheet</p>
+<h2>What is they name of the tag that comes first on any web page?</h2>
+<p>An opening tag</p>
```
Comment (yeeyang) [inline week1/SkylarWilliams]: Was looking for the `<!doctype>` or the `<html>` for the correct answer.

File: week1/SkylarWilliams [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+<h2>How can you see the HTML of a page in your browser?</h2>
+<p>View menu, select source view source.</P>
+<h2>What is semantic markup?</hs>
+<p>Extra infromation; such as where emphasis is placed in a sentence, that something you have written is a quotation.</P>
+<h2>How many types of headlines are there?</h2>
+<p>Six</p>
+<h2>What happens if you use multiple spaces?</h2>
+<p>It will read as one space</p>
+<h2>Name 3 types of lists and describe each one.</h2>
+<P> Ordered list, Unordered list, and definition list.</p>
```
Comment (yeeyang) [inline week1/SkylarWilliams]: It'll still work but you used a capital 'P' for your `<p>` tag. Try to keep your tags consistent.

File: week1/SkylarWilliams [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+<ol>
+<li>Ordered list - List where each item in the list is numbered.</li>
+<li>Unordered list - are list that begin with a bullet point.</li>
+<li>Deffinition list - Are made up of a set of terms along with the deffinitions of each of those terms.</ol>
```
Comment (yeeyang) [inline week1/SkylarWilliams]: Forgot the closing `</li>` for the last list item.

**Comment by yeeyang** [comment]
Good work this week Skylar. Just some minor corrections but overall, the code looks pretty good. Keep it up.

**Pull request closed without merging**